Documentation/ceph-csi-drivers.md

```diff
@@ -92,3 +92,49 @@ kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replicatio
 2. Enable the volume replication controller:
    - For Helm deployments see the [csi.volumeReplication.enabled setting](helm-operator.md#configuration).
    - For non-Helm deployments set `CSI_ENABLE_VOLUME_REPLICATION: "true"` in operator.yaml
+
+## generic ephemeral volume support
+
+The generic ephemeral volume feature adds support for specifying PVCs in the
+`volumes` field to indicate a user would like to create a Volume as part of the pod spec.
+This feature requires the GenericEphemeralVolume feature gate to be enabled.
```

Comment:
Since it's beta now, how about we just say 1.21 or greater is required? If they want to know more about enabling the feature gate, they can find that detail in the k8s doc link at the end of this section.
```suggestion
```